packages/stylable/src/selector-utils.ts

```diff
@@ -74,22 +75,47 @@ export function createChecker(types: Array<string | string[]>) {
     };
 }
 
-export function createRootAfterSpaceChecker() {
-    let hasSpacing = false;
+export function isSpacing(node: SelectorAstNode) {
```

Comment:
maybe we don't need these helper function functions. only isGlobal is useful here